contrib/fastfetch/template.py

```diff
@@ -0,0 +1,63 @@
+pkgname = "fastfetch"
+pkgver = "2.8.10"
+pkgrel = 0
+build_style = "cmake"
+configure_args = ["-DBUILD_TESTS=ON"]
+hostmakedepends = ["cmake", "ninja", "pkgconf"]
+makedepends = [
+    "chafa-devel",
+    "dbus-devel",
+    "dconf-devel",
+    "ddcutil-devel",
+    "libmagick-devel",
+    "libpulse-devel",
+    "libxrandr-devel",
+    "networkmanager-devel",
+    "ocl-icd-devel",
+    "vulkan-headers",
+    "vulkan-loader-devel",
+    "wayland-devel",
+    "xfconf-devel",
+    "zlib-devel",
+]
+pkgdesc = "Fast neofetch-like system information fetching tool"
+maintainer = "Jami Kettunen <jami.kettunen@protonmail.com>"
+license = "MIT"
+url = "https://github.com/fastfetch-cli/fastfetch"
+source = f"{url}/archive/refs/tags/{pkgver}.tar.gz"
+sha256 = "e3ea8388d79cc02ed1a6e45b8e5fec4068507ca03923386bdc9a7b3ff02d2a52"
+tool_flags = {"CFLAGS": ["-DNDEBUG"]}
+# CFI dies immediately (ffPlatformPathAddHome at FFlist.c:31:12)
+hardening = ["vis", "!cfi"]
+
+
+@subpackage("fastfetch-full")
```

Comment:
i don't much see the point of this package?

Reply:
I guess most people can install the extra stuff like `ddcutil` themselves if they want display backlight level infos and whatnot (and almost everything else is automatically installed on a full desktop environment setup anyway). 

but should `lscpu` for example be in `depends` since it's a fallback to determine proper CPU model on some ARM/RISC-V systems? https://github.com/fastfetch-cli/fastfetch/blob/2.8.10/src/detection/cpu/cpu_linux.c#L276-L347

Reply:
lscpu seems ok